recipes/godot_headers/all/conandata.yml

```diff
@@ -0,0 +1,10 @@
+sources:
+  "3.0":
+    url: "https://github.com/godotengine/godot_headers/archive/3.0.zip"
+    sha256: "d73f6dc4d060cadc6c37a82f162cbc0e5ba91db6998ad4d25bbd433184723a00"
+  "3.1":
+    url: "https://github.com/godotengine/godot_headers/archive/3.1.zip"
+    sha256: "95ccd7e8b5809f12f048e82f52d59fee054d8e0ed22fd0b40ce6feb00dc09d6b"
+  "3.2":
+    url: "https://github.com/godotengine/godot_headers/archive/3.2.zip"
```

Comment:
These are archives of top of branches, no releases.
This will work as long as godot doesn't backport a fix.
Does godot do this? If so, maybe use a particular git commit.

Reply:
I was thinking about that. Generally these *should* be in sync, however i believe using the commit archives based on the submodule commits in `godot-cpp` would probably be wise. I just wasn't quite sure if that would be acceptable. I will change that accordingly.

Reply:
applied in latest changeset